migrations/versions/rev-2020-12-28-11:07:56-7374b18af581_.py

```diff
@@ -17,8 +17,8 @@
 
 
 def upgrade():
-    op.execute("UPDATE custom_forms SET is_public=True WHERE form='session';")
+    op.execute("UPDATE custom_forms SET is_public=True WHERE form='session' and (field_identifier='title' or field_identifier='short_abstract' or field_identifier='track' or field_identifier='session_type' or field_identifier='language' or field_identifier='slides' or field_identifier='video');")
     op.execute("UPDATE custom_forms SET is_public=True WHERE form='speaker' and (field_identifier='name' or field_identifier='photo' or field_identifier='country' or field_identifier='short_biography' or field_identifier='website' or field_identifier='facebook' or field_identifier='github' or field_identifier='twitter' or field_identifier='linkedin');")
-    op.execute("UPDATE custom_forms SET is_public=True WHERE form='attendee';")
+    op.execute("UPDATE custom_forms SET is_public=False WHERE form='attendee';")
```

Comment:
It is already false, so let it be

Add downgrade statements as well please

Reply:
ok

Reply:
> Add downgrade statements as well please

same statements in downgrade?